From 7962008be23de99d3bd69e1603d69749f4678a3c Mon Sep 17 00:00:00 2001
From: Jim Stichnoth <jstichnoth@mythtv.org>
Date: Sat, 25 Apr 2015 06:56:51 -0700
Subject: [PATCH] Subtitles: Yet another state calculation fix.

0925cc05806a057b6549a3471ee043d68948d0bd doesn't count AV subtitles as
"text" subtitles, but it should.

(cherry picked from commit 017cc5c8bb66109500e2c4ff771abcac4f87c034)
---
 mythtv/libs/libmythtv/mythplayer.h |    1 -
 1 file changed, 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/mythplayer.h b/mythtv/libs/libmythtv/mythplayer.h
index 3ce2da2..a1df5f2 100644
--- a/mythtv/libs/libmythtv/mythplayer.h
+++ b/mythtv/libs/libmythtv/mythplayer.h
@@ -78,7 +78,6 @@ enum
     kDisplayAllCaptions         = 0x0FF,
     kDisplayTeletextMenu        = 0x100,
     kDisplayAllTextCaptions     = ~kDisplayDVDButton &
-                                  ~kDisplayAVSubtitle &
                                    kDisplayAllCaptions,
 };
 
-- 
1.7.10.2

